src/lib.rs

```diff
@@ -1920,7 +1919,9 @@ unsafe impl<'a> Allocator for &'a Bump {
         new_layout: Layout,
     ) -> Result<NonNull<[u8]>, AllocError> {
         Bump::grow(self, ptr, old_layout, new_layout)
-            .map(|p| NonNull::slice_from_raw_parts(p, new_layout.size()))
+            .map(|p| unsafe {
+                NonNull::new_unchecked(ptr::slice_from_raw_parts_mut(p.as_ptr(), new_layout.size()))
+            })
```

Comment:
Why these changes? Seems unnecessary?

Reply:
To remove all unstable rust features. Except that I had to put one back, so this one may stay as well I guess